Lib/test/test_ntpath.py

```diff
@@ -661,6 +661,8 @@ def test_ismount(self):
         self.assertTrue(ntpath.ismount(b"\\\\.\\c:\\"))
         self.assertTrue(ntpath.ismount(b"\\\\.\\C:\\"))
 
+        self.assertFalse(ntpath.ismount("C:\\NotExist"))
```

Comment:
The drive also needs to not exist - this check already passes on most machines without your fix.

I suggest something like:
```python
for drive in "DEFGHIJKLMNOPQRSTUVWXYZ":
    if not ntpath.exists(drive + ":\\"):
        self.assertFalse(ntpath.ismount(drive + ":\\NotExist"))
        break
else:
    if support.verbose:
        print("No missing drive found to test 'ismount'")
```

Reply:
This is smart and reliable.